### PR TITLE
fix(#185): [Adityan] fix the get UIN/VID button not working in iOS

### PR DESCRIPTION
**Pull request by vijay151096**
issue - 185 -  Unable to get UIN/VID using AID.

It was not particularly working in iOS devices. It was mentioned that iOS does not support loading of one modal over another.
I made a fix for that by adding a condition.

### CODE REVIEW

File: screens/Home/MyVcs/GetVcModal.tsx [hunk 1]
```diff
@@ -13,7 +13,9 @@ export const GetVcModal: React.FC<GetVcModalProps> = (props) => {
     <React.Fragment>
       <GetIdInputModal
         service={props.service}
-        isVisible={true}
+        isVisible={
+          !controller.isAcceptingOtpInput && !controller.isRequestingCredential
```
Comment (ravikp) [inline screens/Home/MyVcs/GetVcModal.tsx:17]: Can we look if this extraction of the logic is possible `stateInAcceptingApplicationID() -> bool`? It looks like the logic is being formulated based on deny list of states, rather it is better to show the dialog based on allowed state.

**Review by ravikp — CHANGES_REQUESTED** [review]
Check if re-writing the logic based on the allowed list of states possible.

File: screens/Home/MyVcs/GetVcModal.tsx (hunk 1 above)
Comment (adityankannan-tw) [reply screens/Home/MyVcs/GetVcModal.tsx]: we have used the "isAcceptingUinInput" state to load the modal, tested it and working as expected.

**Comment by tilak-puli** [comment]
looks good to me.

**Pull request merged**